src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java

```diff
@@ -28,19 +28,18 @@ public void onPrefixChange(PrefixChangeEvent event) {
     if (event.getUUID() == null) {
       return;
     }
-    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(event.getUUID());
-    if (matchPlayer == null
-        || matchPlayer.getBukkit() == null
-        || matchPlayer.getBukkit().isOnline()) {
+    final Player player = Bukkit.getPlayer(event.getUUID());
+    if (player == null || !player.isOnline()) {
       return;
     }
```

Comment:
Why is the `Bukkit#getPlayer` check nessecary, can't you just use the `MatchManager#getPlayer` check?